src/H5C.c

```diff
@@ -7275,9 +7275,9 @@ H5C__load_entry(H5F_t *f,
                                 (mpi_code = MPI_Bcast(image + len, buf_size, MPI_BYTE, 0, comm)))
                                 HMPI_GOTO_ERROR(NULL, "MPI_Bcast failed", mpi_code)
                         } /* end if */
-#endif                    /* H5_HAVE_PARALLEL */
-                    }     /* end if */
-                }         /* end if (actual_len != len) */
+#endif /* H5_HAVE_PARALLEL */
+                    } /* end if */
+                }     /* end if (actual_len != len) */
```

Comment:
Comments at the end of braces shouldn't be lined up. We'll need to fix the formatter to stop doing this.

Reply:
> Comments at the end of braces shouldn't be lined up. We'll need to fix the formatter to stop doing this.

Yeah, not my favorite either.   Fixing the formatter to avoiding doing this would be nice.